client/src/Molecules/ContestCard/ContestCard.js

```diff
@@ -47,30 +61,61 @@ const ContestCard = ({ contestData }) => {
       },
     });
   };
-
+  const imgs = [
+    dog_1,
+    dog_2,
+    dog_3,
+    dog_4,
+    dog_5,
+    dog_6,
+    dog_7,
+    dog_8,
+    dog_9,
+    dog_10,
+    dog_11,
+    dog_12,
+  ];
+  let i = Math.floor(Math.random() * imgs.length);
```

Comment:
Maybe it would be more transparent if You extract this to an outside function in tools folder  ?

Reply:
I don't think it's necessarily (function in tools). Ok, I can change name